src/Dict.cc

```diff
@@ -18,8 +18,8 @@ TEST_SUITE_BEGIN("Dict");
 
 struct ReportDuration
 	{
-	std::chrono::_V2::system_clock::time_point start;
-	std::chrono::_V2::system_clock::time_point stop;
+	std::chrono::system_clock::time_point start;
```

Comment:
This still isn't it unfortunately. macOS complains as below.

```
[ 93%] Building CXX object src/CMakeFiles/zeek_objs.dir/EventTrace.cc.o
../../src/Dict.cc:26:9: error: no viable overloaded '='
                start = std::chrono::high_resolution_clock::now();
                ~~~~~ ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/c++/v1/__chrono/time_point.h:31:28: note: candidate function (the implicit copy assignment operator) not viable: no known conversion from 'time_point<std::chrono::steady_clock, duration<[...], ratio<[...], 1000000000>>>' to 'const time_point<std::chrono::system_clock, duration<[...], ratio<[...], 1000000>>>' for 1st argument
class _LIBCPP_TEMPLATE_VIS time_point
                           ^
```

Reply:
Hi, Tim,

Let me back track the PR to just include the Dict.h modifications and exclude the unit tests. This make it easy and we need to verify it.

I'll send another PR for unit test